R/FilteredData.R

```diff
@@ -759,6 +801,18 @@ FilteredData <- R6::R6Class( # nolint
             }
           )
 
+          private$filter_panel_ui_id <- session$ns(NULL)
+          observeEvent(
+            eventExpr = input[["filter_turn_onoff"]],
+            handlerExpr = {
+              if (isTRUE(input[["filter_turn_onoff"]])) {
+                self$filter_panel_enable()
+              } else {
+                self$filter_panel_disable()
+              }
```

Comment:
I would suggest to put all of `shinyjs::disable/enable` calls here and use save/restore methods. This would keep `shiny` functionalities nicely decoupled - methods relates only to _data fields_ and `shiny` parts lives inside the modules.
Moreover, I do not see (and I couldn't imagine) other place where we would call `self$filter_panel_enable/disable()`.

Reply:
On another side - please put logger entries

Reply:
"other place where we would call" - DRY rule is not the only reason to separate the code. Nice that you are raising it, please check how overcrowded is the server of filter panel now. WE should work to make it more clear and readable.

Reply:
> DRY rule is not the only reason to separate the code

Yes you are definitely right here but my first point remains.
Long time ago we have a lengthy discussions with @gogonzo and Konrad about decoupling shiny-related parts from the class definitions. Unfortunately there is no good design pattern and we decided to still keep it inside the class definition but only inside one module method or two ui/server methods. This indeed means that those the method(s) definitions would be lengthy but it also assures nice decoupling of class' methods from the shiny part which should serve just as an add-on which calls those methods. Separate shiny-methods would break that and make the code less separated.